agent/bench-scripts/pbench-fio.md

```diff
@@ -200,21 +200,21 @@ Then you construct a fio job file for your initial sequential tests, and this wi
 
     [global]
     # size of each FIO file (in MB)
-    size=$@
+    size=1GiB
     # do not use gettimeofday, much too expensive for KVM guests
     clocksource=clock_gettime
     # give fio workers some time to get launched on remote hosts
     startdelay=5
     # files accessed by fio are in this directory
-    directory=$@
+    directory=/mnt/fio/files
     # write fio latency logs in /var/tmp with "fio" prefix
-    write_lat_log=/var/tmp/fio
+    write_lat_log=fio
```

Comment:
Why do we need to change this file name and location without also changing the comment?

Reply:
With the absolute path provided, the /var/lib/pbench-agent/fio__2020.05.08T08.24.48/1-write-4KiB/sample1/result.json is not created. 
Logs:
running fio job: /var/lib/pbench-agent/fio__2020.05.08T08.24.48/1-write-4096KiB/fio.job (sample1)
Warning: ALREADY_ENABLED: '8765:tcp' already in 'public'
Warning: ALREADY_ENABLED: '8765:tcp' already in 'public'
fio job complete
fio-postprocess: could not find any result files to process, exiting